.travis.yml

```diff
@@ -57,13 +57,29 @@ matrix:
       - MASTER=spark://127.0.0.1:7077
     install:
       - make build
+      - 
     script:
+      # start dependencies
       - ./scripts/get_apache_spark.sh "2.2.0" "2.7" || travis_terminate 1
       - ./scripts/start_apache_spark_cluster.sh "127.0.0.1" "7077" || travis_terminate 1
+      # python will be installed after https://github.com/src-d/gemini/issues/60
+      # and after merge https://github.com/src-d/gemini/pull/98 we can use local fe runner
+      - docker build -t srcd/gemini-fe -f FE.Dockerfile .
+      - ./scripts/start_docker_fe.sh
       - sleep 10 # wait for it doesn't work, port is avalable but scyllaDB is still down
       - ./hash src/test/resources/siva || travis_terminate 1
+      # query without similarity
       - ./query ./src/test/resources/LICENSE
+      # query with similarity
```

Comment:
`#Load hashtables from apollo DB dump`

Reply:
it's obvious from the command that it loads dump into DB, no matter actually where the fixture comes from.
I added comments to split groups of commands by what they do, not rewrote the command in English.